geoutils/raster/array.py

```diff
@@ -90,8 +90,8 @@ def get_xy_rotated(raster: gu.Raster, along_track_angle: float) -> tuple[NDArray
 
     myang = np.deg2rad(along_track_angle)
 
-    # Get grid coordinates
-    xx, yy = raster.coords(grid=True)
+    # Get grid coordinates (only relative is important, we don't care about offsets, so let's fix lower-left)
+    xx, yy = raster.coords(grid=True, force_offset="ll")
```

Comment:
Idf we don't care, why set `force_offset` and not just use the default?

Reply:
The tests were written to check in comparison to a rotation of an array matching values with starting 0,0 coordinate (lower-left), which is why I kept it! Don't want to re-write all the tests.
Added a comment to further clarify :smile: